fmpp/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
```

Comment:
This is a submodule. tools is not right now, do you want to make tools part of the project module hierarchy?